applications/plugins/SofaMatrix/src/SofaMatrix/FillReducingOrdering.inl

```diff
@@ -41,6 +44,8 @@ FillReducingOrdering<DataTypes>::FillReducingOrdering()
     addOutput(&d_position);
     addOutput(&d_hexahedra);
     addOutput(&d_tetrahedra);
+
+    sofa::helper::getWriteAccessor(d_orderingMethod)->setSelectedItem(0);
```

Comment:
I wonder if there is no room for a PR to make this writing a bit more elegant (not in this PR).